src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -123,23 +123,19 @@ std::string jsonRequestTreat
     break;
 
   case SubscriptionsRequest:
+    answer = parseSubscription(ciP, &parseDataP->subsV2);
+    if (answer != "OK")
     {
-
-      answer = parseSubscription(ciP, &parseDataP->subsV2);
-      if (answer != "OK")
-      {
-        return answer;
-      }
+      return answer;
     }
     break;
 
   case IndividualSubscriptionRequest:
+    answer = parseSubscription(ciP, &parseDataP->subsV2, true);  // NOTE: partial == true
+    LM_W(("httpInfo.extended: %s", parseDataP->subsV2.notification.httpInfo.extended? "true" : "false"));
+    if (answer != "OK")
     {
-      answer = parseSubscription(ciP, &parseDataP->subsV2, true);  // NOTE: partial == true
-      if (answer != "OK")
-      {
-        return answer;
-      }
+      return answer;
     }
     break;
 
```

Comment:
In this case it is a little fragment... however, not paying attention to style change along with "real" changes could make that the later get lost to the reviewer in a sea of red-green noise.

We should find a solution to this... it is happening time after time. I have some ideas (to be discussed at skype).